packages/webapp/src/pages/members/[id].tsx

```diff
@@ -2,23 +2,29 @@ import { GetServerSideProps } from "next";
 import { QueryClient, useQuery } from "react-query";
 import { dehydrate } from "react-query/hydration";
 
-import { CallToAction, Card, RawLayout, SingleColLayout } from "_app";
 import {
-    getMember,
-    MemberCard,
-    MemberCollections,
-    MemberHoloCard,
-} from "members";
-
-const QUERY_MEMBER_DATA = "query_member_data";
+    CallToAction,
+    Card,
+    queryMemberData,
+    RawLayout,
+    SingleColLayout,
+} from "_app";
+import { MemberCard, MemberCollections, MemberHoloCard } from "members";
 
+/**
+ * We have an issue if the member is not found in the development environment
+ * due to dehydration with JSON not being able to serialize `undefined`:
+ * Error: Error serializing `.dehydratedState.queries[0].state.data`
+ * returned from `getServerSideProps` in "/members/[id]".
+ * Reason: `undefined` cannot be serialized as JSON. Please use `null`
+ * or omit this value.
+ * Let's track this here: https://github.com/tannerlinsley/react-query/issues/1978
+ */
```

Comment:
I noticed this before. While it fails loudly in development mode, in production, it thankfully falls back gracefully to our "member not found" screen.